src/components/atmosphere.jl

```diff
@@ -51,6 +59,18 @@ Interfacer.step!(sim::HeatEquationAtmos, t) = Interfacer.step!(sim.integrator, t
 Interfacer.reinit!(sim::HeatEquationAtmos) = Interfacer.reinit!(sim.integrator)
 
 get_field(sim::HeatEquationAtmos, ::Val{:T_atm_sfc}) = sim.integrator.u[1]
-function update_field!(sim::HeatEquationAtmos, ::Val{:T_oce_sfc}, field)
-    parent(sim.integrator.p.T_sfc)[1] = field
+function update_field!(sim::HeatEquationAtmos, field_1, field_2)
+    if sim.params.boundary_mapping == "mean"
+        parent(sim.integrator.p.T_sfc)[1] = field_1
+        parent(sim.integrator.p.T_ice)[1] = field_2
+    else
+        parent(sim.integrator.p.T_sfc) .= field_1
```

Comment:
Using the "CIT" mapping option does not work due to this line: `coupled_heat_equations(boundary_mapping="cit")` crashes with a `DimensionMismatch` error. I have double checked and `sim.integrator.p.T_sfc` has the dimension of the whole vertical CenteredFiniteDifference space, while `field_1` has two values (I guess corresponding to two time steps).

Reply:
Fixed in [94523ec](https://github.com/valentinaschueller/clima-playground/pull/11/commits/94523ecbaabfa7266ec83babcf3f9cb809ea3703)